src/ocamlorg_frontend/pages/package_versions.eml

```diff
@@ -0,0 +1,44 @@
+let render (package : Package.package)
+=
+Layout.render
+~title:(Printf.sprintf "%s Versions" package.name)
+~description:"Package Versions" @@
+<div class="bg-default dark:bg-dark-default py-8">
+  <div class="container-fluid flex items-center gap-2">
+    <a href="/p/<%s package.name %>" class="h-12 w-12 m-5 border-primary-600 border-2 rounded-full flex items-center justify-center"><%s! Icons.arrow_left "h-8 w-8 text-primary-600"; %></a>
+    <h1 class="text-xl font-bold"><%s package.name %> Versions </h1>(x)
+  </div>
+  <div class="container-fluid flex justify-center">
+    <table class="table-auto border-separate border-spacing-y-6">
+        <thead class="container-fluid text-md text-lighter text-left">
+            <tr class="py-4">
+                <th class="text-left font-normal border-b-2 border-gray-200 py-4 pr-12 pl-3"><%s! Icons.bars_3 "h-5 w-5"; %></th>
+                <th class="text-left font-normal border-b-2 border-gray-200 pr-12">Version</th>
+                <th class="text-left font-normal border-b-2 border-gray-200 pr-12">Release Date</th>
+                <th class="text-left font-normal border-b-2 border-gray-200 pr-12 pl-5">Links</th>
+            </tr>
+        </thead>
+        <tbody>
+          <% package.versions |> List.iter (fun (version : string) -> %>
+          <tr class="pt-5 tr-selected">
```

Comment:
Never do padding on surrounding elements when you can do the padding on the `<a>` tag.

The click area of a link should always be as big as the visuals of the link.

When there's white space around links, it's always worth to consider if that can be subsumed into the link's padding so that the click area is bigger - assume that some people are very clumsy and have a hard time clicking/tapping on things or they just have big fingers.

In this particular case, the padding should be on the link.